docs/source/Htypes.rst

```diff
@@ -0,0 +1,5 @@
+Htypes
+======
+
+.. currentmodule:: hub.htype
```

Comment:
I think instead of the table that we have currently, some more detail about each htype could be useful